addons/event_booth_sale/tests/test_event_booth_sale.py

```diff
@@ -72,3 +73,28 @@ def test_event_booth_prices_with_sale_order(self):
                          "Untaxed amount should be the sum of the booths prices ($200.0).")
         self.assertEqual(float_compare(sale_order.amount_total, 220.0, precision_rounding=0.1), 0,
                          "Total amount should be the sum of the booths prices with 10% taxes ($200.0 + $20.0).")
+
+        # Confirm the SO.
+        sale_order.action_confirm()
+
+        # Create and check that the invoice was created
+        invoice = sale_order._create_invoices()
+        self.assertEqual(len(sale_order.invoice_ids), 1, "Invoice not created.")
+
+        # Confirm the invoice and check SO invoice status
+        invoice.action_post()
+        self.assertTrue(sale_order.invoice_status == 'invoiced', 'order is not invoiced')
+
+        # Add group `group_account_invoice` to user to allow to pay the invoice
+        self.env.user.groups_id += self.env.ref('account.group_account_invoice')
```

Comment:
Could be done at beginning of test (you generally don't gain privileges between two calls :) ) .